lib/public/views/lhcPeriods/format/formatDistinctLhcBeamEnergies.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+
+const distinctEnergiesProcessingConfiguration = { // FOR NOW HARDCODED
+    wellDefinedEnergyLevelToDisplay: {
+        450: 450,
+        6800: 6800,
+        7000: 7000,
+        2680: '5360/2',
+    },
+    acceptableEnergyMargin: 0.01,
+};
```

Comment:
If you don't need it yet, don't put it! Use constants when you can.
I would rather do:

```js
const WELL_DEFINED_ENERGY_LEVELS = [450, 2680, 6800, 7000]; // Sorted!
const ENERGY_LEVELS_CUSTOM_NAME = {2680: '5360/2'};
const ENERGY_LEVEL_MARGIN = 0.01;
const DISPLAY_PRECISION = 2;
```

Then: 
```js
getClosestEnergyLevel(lhcBeamEnergy) { // No need for the second argument, if it's needed at some point simply add it then
    for (const energyLevel of WELL_DEFINED_ENERGY_LEVELS) {
        if (lhcBeamEnergy < energyLevel - ENERGY_LEVEL_MARGIN) {
            return null; // next energy level is out of reach
        } else if (lhcBeamEnergy <= energyLevel + ENERGY_LEVEL_MARGIN) {
            return energyLevel;
        }
        // There may be other energy levels in reach, continue looping
    }
    return null;
}
```

Then to display, you simply do 
```js
const energyLevel = getClosestEnergyLevel(lhcBeamEnergy);
const energyToDisplay = energyLevel !== null ? energyLevel : lhcBeamEnergy;
energyToDisplay in ENERGY_LEVELS_CUSTOM_NAME ? ENERGY_LEVELS_CUSTOM_NAME[energyToDisplay] : energyToDisplay.toFixed(DISPLAY_PRECISION);
```

Reply:
applied